### ISSUE TITLE
Errors on colors.lua:41: invalid key: create

### ISSUE THREAD

**Issue by mortymacs** [Bekaboo/deadcolumn.nvim#17]
After upgrading my nvim packages, I got this:

```
Error detected while processing /home/mort/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/mort/.config/nvim/lua/theme.lua:3: /home/mort/.config/nvim/init.lua..ColorScheme Autocommands for "*": Vim(append):Error executing lua callback:
...hare/nvim/lazy/deadcolumn.nvim/lua/deadcolumn/colors.lua:41: invalid key: create
stack traceback:
        [C]: in function 'nvim_get_hl'
        ...hare/nvim/lazy/deadcolumn.nvim/lua/deadcolumn/colors.lua:41: in function 'get'
        ...local/share/nvim/lazy/deadcolumn.nvim/lua/deadcolumn.lua:17: in function 'get_hl_hex'
        ...local/share/nvim/lazy/deadcolumn.nvim/lua/deadcolumn.lua:24: in function <...local/share/nvim/lazy/deadcolumn.nvim/lua/deadcolumn.lua:23>
        [C]: in function 'colorscheme'
        /home/mort/.config/nvim/lua/theme.lua:3: in main chunk
        [C]: in function 'require'
        /home/mort/.config/nvim/init.lua:8: in main chunk
stack traceback:
        [C]: in function 'colorscheme'
        /home/mort/.config/nvim/lua/theme.lua:3: in main chunk
        [C]: in function 'require'
        /home/mort/.config/nvim/init.lua:8: in main chunk
```

Just installing the package without any config.

My system:
```
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.1693350652
```

**Comment by Bekaboo** [comment]
Ah I think that's a new key introduced in 0.10.0 nightly, should be a simple fix.

**Comment by Bekaboo** [comment]
Should be fixed in commit b84cdf2, can you confirm?

**Comment by mortymacs** [comment]
Thank you!
I just tested and it works fine!
